cgi/product_multilingual.pl

```diff
@@ -56,6 +56,11 @@
 
 $debug = 1;
 
+if ($User_id eq 'unwanted-user-french') { 
```

Comment:
Unrelated to #1145 though? Committed to wrong branch?

Reply:
Yes, in fact we needed 1145 live in production very quickly (some producer complaints), so it's already live in production now. But then I needed more changes live on production, like this one, that's why it got mixed up. My bad.